### ISSUE TITLE
Vocab not working as expected; perhaps not at all?

### ISSUE THREAD

**Issue by nostodnayr** [vale-cli/vale#227]
Affected: Vale 2.3.2 on Mac OS and Linux

I’m testing the vocabulary feature using [vale-boilerplate](https://github.com/errata-ai/vale-boilerplate) as a starting point. I added a Vocab line to the config and then added a term to the accept (extremely) and reject (basic) lists.

When I run `vale README.md`, the output is:

	 README.md
	 13:20   warning  'extremely' is a weasel word!  write-good.Weasel
	 15:120  warning  'However' is too wordy.        write-good.TooWordy
	 27:6    warning  'is' is repeated!              write-good.Illusions

	✖ 0 errors, 3 warnings and 0 suggestions in 1 file.

I expected to see an error for ‘basic’ and to not see the warning for ‘extremely’.

I’m not sure if I’m misunderstanding how the feature works (I’m entirely new to Vale), or if it’s malfunctioning.

The diff:

	diff --git a/.vale.ini b/.vale.ini
	index e0e8980..d278cd9 100644
	--- a/.vale.ini
	+++ b/.vale.ini
	@@ -1,4 +1,5 @@
	 StylesPath = styles
	+Vocab = blog

	 [*.md]
	 BasedOnStyles = write-good
	diff --git a/styles/Vocab/Blog/accept.txt b/styles/Vocab/Blog/accept.txt
	index e69de29..03c0370 100644
	--- a/styles/Vocab/Blog/accept.txt
	+++ b/styles/Vocab/Blog/accept.txt
	@@ -0,0 +1 @@
	+extremely
	diff --git a/styles/Vocab/Blog/reject.txt b/styles/Vocab/Blog/reject.txt
	index e69de29..15a13db 100644
	--- a/styles/Vocab/Blog/reject.txt
	+++ b/styles/Vocab/Blog/reject.txt
	@@ -0,0 +1 @@
	+basic

**Comment by jdkato** [comment]
There are two problems:

1. The built-in [vocab-related rules](https://docs.errata.ai/vale/vocab) are part of the `Vale` style ( `Vale.Terms` and `Vale.Avoid`). To see results from these rules, you need to enable the `Vale` style: `BasedOnStyles = Vale, write-good`.

2. `accept.txt` currently only applies to rules that support exceptions (i.e., those that extend `spelling`, `capitalization`, or `conditional`). So, it won't apply to a rule like `write-good.Illusions`.

**Comment by nostodnayr** [comment]
> The built-in vocab-related rules are part of the Vale style ( Vale.Terms and Vale.Avoid). To see results from these rules, you need to enable the Vale style: BasedOnStyles = Vale, write-good.

Ah! This was the most unclear part, as there was no mention of it in the docs.

Thanks for the additional information; I think it's working as expected now.

**Comment by wburningham** [comment]
Should the `Vale.Spelling` style ignore words that are in the accept vocab?
